.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,33 @@
+name: "CodeQL"
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]
+        # CodeQL currently supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python', 'ruby' ]
+        # Modified to support codebase. Currently no support for Solidity Language
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Init CodeQL
+      uses: github/codeql-action/init@v1
+      with:
+        languages: ${{ matrix.language }}
+
+    - name: Perform CodeQL Analysis
+      uses: github/codeql-action/analyze@v1
```

Comment:
CodeQL v1 is being deprecated by the 18th of this month.

Reply:
Updated version.